libkpod/kill.go

```diff
@@ -0,0 +1,32 @@
+package libkpod
+
+import (
+	"github.com/docker/docker/pkg/signal"
+	"github.com/kubernetes-incubator/cri-o/oci"
+	"github.com/kubernetes-incubator/cri-o/utils"
+	"github.com/pkg/errors"
+	"os"
+)
+
+// ContainerKill kills ones or more containers with a user provided signal
+func (c *ContainerServer) ContainerKill(container string, signalString string) (string, error) {
+
+	ctr, err := c.LookupContainer(container)
+	if err != nil {
+		return "", errors.Wrapf(err, "failed to find container %s", container)
+	}
+	cStatus := c.runtime.ContainerStatus(ctr)
+
+	// Check if the signalString provided by the user is valid
+	// Invalid signals will return err
+	if _, err = signal.ParseSignal(signalString); err != nil {
+		return "", err
+	}
+	if cStatus.Status != oci.ContainerStateStopped {
```

Comment:
Should we check more status, is there other status like it was never started or is this also stopped?

Reply:
it looks like for docker, they check exclusively for running and error when not so ... ill follow that.